### ISSUE TITLE
Can't record audio without also playing it

### ISSUE THREAD

**Issue by alito** [mgbellemare/Arcade-Learning-Environment#116]
The new recording options are very cool. Setting display_screen to False, and record_screen_dir, it's  possible to record the video without watching it.

The same does not seem to be possible with sound though. If you don't set sound to True, 'record_sound_filename' doesn't save the sound. It'd be good if that'd be possible. (Not sure if this is a bug or an enhancement. It's inconsistent with the behaviour of video)

**Comment by mgbellemare** [comment]
Yes, sound is still a work in progress. Part of the issue is that currently sound recording is subordinate to SDL events. There is a solution, but it requires a decent engineering effort -- we're looking for someone with a little spare time on their hands to wrangle that beast :)
